jobManager.py

```diff
@@ -44,32 +44,48 @@ def run(self):
 
     def __manage(self):
         self.running = True
+        # job-associated instance id
+        nextId = 10000
```

Comment:
where does this 10000 number come from? should this be `Config.MAX_JOBID` instead?